corefxlab-base.yml

```diff
@@ -46,7 +46,7 @@ jobs:
                 /p:DotNetArtifactsCategory=$(_DotNetArtifactsCategory)
                 /p:OfficialBuildId=$(BUILD.BUILDNUMBER)          
             - name: _buildArguments
-              value: -Configuration $(_BuildConfig) -ci -sign /p:BuildPackages=true -pack -publish $(_InternalBuildArgs)
+              value: -Configuration $(_BuildConfig) -ci -sign /p:BuildPackages=true -pack -publish -test $(_InternalBuildArgs)
```

Comment:
is the `BuildPackages` property necessary?

Reply:
This is going to add testing to the official build, do we want that? I would remove it from here as in official builds we usually don't run tests.

Reply:
Oh, I didn't know that. Ok